nixos/modules/services/backup/btrbk.nix

```diff
@@ -51,6 +51,10 @@ let
     '';
 in
 {
+  imports = [
+    (mkRenamedOptionModule [ "services" "btrbk" "niceness" ] [ "services" "btrbk" "daemonNiceLevel" ])
```

Comment:
You need to grab this from the `lib` set before using it. I think this causes both tests to fail.